PermissionsEx/permissions.yml

```diff
@@ -2912,6 +2912,35 @@ groups:
         - ch.alias.haste5
         - ch.alias.haste6
         - ch.alias.haste7
+   citizensteel:
```

Comment:
Commander Steel (also Steel or Citizen Steel) is the name of three superheroes published by DC Comics

Henry Heywood
Hank Heywood III
Nathan Heywood
Nathaniel Heywood (different than nathan)
The New 52

further reading
https://en.wikipedia.org/wiki/Commander_Steel
http://dc.wikia.com/wiki/Nathaniel_Heywood_(New_Earth)
http://dc.wikia.com/wiki/Nathan_Heywood_(Arrow)